nemo/collections/tts/torch/tts_data_types.py

```diff
@@ -67,6 +67,14 @@ class LMTokens(TTSDataType):
     name = "lm_tokens"
 
 
+class GSTRefAudio(TTSDataType, WithLens):
```

Comment:
Let's rename this to `ReferenceAudio` and name = "reference_audio". So, we do not couple this with GST only.